packages/apps/hr/src/screens/ExpenseListScreen.js

```diff
@@ -0,0 +1,226 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useCallback, useMemo, useState} from 'react';
+import {View, StyleSheet, Dimensions} from 'react-native';
+import {
+  Screen,
+  ScrollList,
+  HeaderContainer,
+  ToggleSwitch,
+  getCommonStyles,
+  useThemeColor,
+  MultiValuePicker,
+  Text,
+} from '@axelor/aos-mobile-ui';
+import {useDispatch, useSelector, useTranslator} from '@axelor/aos-mobile-core';
+import {ExpenseCard} from '../components';
+import {searchExpense} from '../features/expenseSlice';
+import {Expense} from '../types';
+
+const ExpenseListScreen = ({}) => {
+  const I18n = useTranslator();
+  const Colors = useThemeColor();
+  const dispatch = useDispatch();
+
+  const {user} = useSelector(state => state.user);
+  const {loadingExpense, moreLoading, isListEnd, expenseList} = useSelector(
+    state => state.expense,
+  );
+
+  const [mode, setMode] = useState('myExpense');
```

Comment:
🤢 
No magic number or string. This string is used multiples times, create a global variable ! It's easier to maintain !